client/client.go

```diff
@@ -702,6 +702,9 @@ func (c *client) UpdateOption(option DynamicOption, value any) error {
 			return err
 		}
 	case EnableTSOFollowerProxy:
+		if c.getServiceMode() != pdpb.ServiceMode_PD_SVC_MODE {
```

Comment:
Will MaxTSOBatchWaitInterval also need it?

Reply:
No, `MaxTSOBatchWaitInterval` is supported since it only relies on the client implementation.